### PR TITLE
Deal with text shadow

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

**Comment by johngruen** [comment]
@lmorchard this looks like it failed w/o making it very far in circle, and i don't have permissions to retrigger the build

**Comment by chuckharmston** [comment]
screen shot 2018-03-07 at 13 02 52

Looks like it passed from here.

**Comment by lmorchard** [comment]
Looks like it passed... I think CircleCI or npm might be having some issues today

**Comment by johngruen** [comment]
whelp, there you go

File: src/extension/background.js [hunk 1]
```diff
@@ -70,6 +67,14 @@ const applyTheme = ({ theme }) => {
     colors: {}
   };
 
+  // the headerURL is required in < 60,
+  // but it creates an ugly text shadow.
+  // So only add it for older FFs only.
+  const fxVersion = navigator.userAgent.toLowerCase().split("firefox/")[1];
+  if (fxVersion < 60) {
+    newTheme.headerURL = BLANK_IMAGE;
```
Comment (johngruen) [inline src/extension/background.js]: err, this should be newTheme.image.headerURL

Comment (lmorchard) [inline src/extension/background.js]: This should be `newTheme.images.headerURL` - getting an error from this:
```
Error processing headerURL: An unexpected property was found in the WebExtension manifest.
```

**Pull request merged**